<!DOCTYPE html>
<html lang="en">

<head>
  <title>emma_portfolio</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
<!--
  <meta http-equiv='cache-control' content='no-cache'>
  <meta http-equiv='expires' content='0'>
  <meta http-equiv='pragma' content='no-cache'>
-->
  <link rel="icon" href="img/cv_picture4_c2.jpg" type="image/x-icon">

  <link rel="stylesheet" href="css/main.bundle.css">
  <script src="https://kit.fontawesome.com/4e5a72c756.js"></script>
</head>

<body>
  <nav class="navbar is-primary" role="navigation" aria-label="main navigation">
    <div class="navbar-brand">
      <div class="navbar-item is-hidden-desktop">
        <a href="https://github.com/ericart89" class="icon is-large has-text-light">
          <i class="fab fa-2x fa-github"></i>
        </a>
      </div>

      <div class="navbar-item is-hidden-desktop">
        <a href="https://bitbucket.org/EmmaRicart" class="icon is-large has-text-light">
          <i class="fab fa-2x fa-bitbucket"></i>
        </a>
      </div>

    </div>
    <div class="navbar-menu">
      <div class="navbar-end">
        <div class="navbar-item">
          <a href="https://github.com/ericart89" class="icon is-large has-text-light">
            <i class="fab fa-2x fa-github"></i>
          </a>
        </div>

        <div class="navbar-item">
          <a href="https://bitbucket.org/EmmaRicart" class="icon is-large has-text-light">
            <i class="fab fa-2x fa-bitbucket"></i>
          </a>
        </div>

      </div>
    </div>
  </nav>

  <section class="hero is-primary">
    <div class="hero-body">
      <div class="container">
        <div class="level">
          <div class="level-item has-text-centered">
            <figure class="image is-128x128">
              <img class="is-rounded" src="img/cv_picture4_c2.jpg">
            </figure>
          </div>
        </div>
        <div class="columns has-text-centered">
          <div class="column">
            <h1 class="title is-1">
              Emma Ricart Altimiras, PhD
            </h1>
            <h2 class="subtitle">
              Bioinformatician & Software developer
            </h2>
            <p>
              <a href="docs/emma_ricart_altimiras_cv.pdf" class="icon has-text-light">
                <i class="fas fa-file-pdf"></i> <strong>Resume</strong>
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>

  <section class="section">
    <div class="container">
      <h1 class="title">About Me</h1>
      <hr />
      <p>Bioinformatician in the intersection of data analysis, pipeline design and software development.
        In the era of big data, my goal is to formulate data-driven hypothesis and validate them.
        I achieve this by 1) performing statistical analyses, 2) creating tools for data visualization,
        3) developing pipelines or 4) building computational infrastructures such as databases.
        I hold a a PhD in Bioinformatics and I conducted interdisciplinary research working with biologists,
        physicists and software engineers. I'm deeply committed to promoting best programming practices for
        shareable and reproducible research!</p>
    </div>
  </section>
  <section class="section">
    <div class="container">
      <h1 class="title">Experience</h1>
      <hr />
      <div class="card">
        <div class="card-content">
          <div class="media">
            <div class="media-left">
              <figure class="image is-48x48">
                <img src="img/chuv_logo.png" alt="Placeholder image">
              </figure>
            </div>
            <div class="media-content">
              <p class="title is-4"> AGORA Cancer Research Center - Bassani Lab</p>
              <p class="subtitle is-6">Bioinformatician | Database administrator - <time datetime="2021-2">February 2021</time> - <time
                  datetime="2024-2">February 2024</time> </p>
            </div>
          </div>
          <div class="content">
            <p> My responsabilities in Prof. Michal Bassani lab were multiple and diverse. From pipeline and software
              development to data storage and visualization:</p>
            <ul>
              <li>Developed a workflow for large-scale identification and quantification of HLAI/HLAII peptides from mass spectrometry experiments</li>
              <li>Implemented an ETL process for the creation of an immunopeptidomics data warehouse which included +1.000.000 peptides identified with the workflow</li>
              <li>Used the collected data to create a scoring system for the prioritization of neoantigens, directly impacting the clinical trials</li>
              <li>Build a web interface for querying and visualizing the database</li>

            </ul>
            <div class="tags">
              <span class="tag">Python</span>
              <span class="tag">R</span>
              <span class="tag">Bash</span>
              <span class="tag">Nextflow</span>
              <span class="tag">HPC</span>
              <span class="tag">Immunopeptidomics</span>
              <span class="tag">Mass spectrometry</span>
              <span class="tag">Data Visualization</span>
              <span class="tag">Biological Data Analysis</span>
              <span class="tag">Docker</span>
              <span class="tag">Software development</span>
              <span class="tag">Git</span>

            </div>
          </div>
        </div>
      </div>
      <div class="card card-gap">
        <div class="card-content">
          <div class="media">
            <div class="media-left">
              <figure class="image is-48x48">
                <img src="img/sib_logo.png" alt="Placeholder image">
              </figure>
            </div>
            <div class="media-content">
              <p class="title is-4">Swiss Institute of Bioinformatics - Proteome Informatics Group</p>
              <p class="subtitle is-6">Ph. D. Student - <time datetime="2015-9">September 2015</time> - <time
                  datetime="2019-9">September 2019</time></p>
            </div>
          </div>
          <div class="content">
            <p>I was granted with a SIB PhD Fellowship in the group of Frederique Lisacek, a group highly focused on the
            computational side of bioinformatics, where I adquired experience in software development and good programming
            practices:</p>
            <ul>
              <li>Developed rBAN, a java package and web application tool for the retro biosynthesis of non-ribosomal peptides</li>
              <li>Developed NRPro, a web application tool for the identification and annotation of tandem mass spectra from peptidic natural products</li>
              <li>Worked with SMILES format and graph structures for the substructures search and the fragmentation algorithms used by the tools</li>
              <li>Build a NoSQL database for the backend of NRPro integrating data from Norine, ChEBI and Natural Products Atlas</li>
            </ul>
            <div class="tags">
              <span class="tag">Java</span>
              <span class="tag">R</span>
              <span class="tag">Maven</span>
              <span class="tag">MongoDB</span>
              <span class="tag">AngularJS</span>
              <span class="tag">Mass spectrometry</span>
              <span class="tag">Peptidomics</span>
              <span class="tag">Data Visualization</span>
              <span class="tag">Biological Data Analysis</span>
              <span class="tag">Software development</span>
              <span class="tag">Git</span>

            </div>
          </div>
        </div>
      </div>
      <div class="card card-gap">
        <div class="card-content">
          <div class="media">
            <div class="media-left">
              <figure class="image is-48x48">
                <img src="img/jax_logo3.png" alt="Placeholder image">
              </figure>
            </div>
            <div class="media-content">
              <p class="title is-4">JAX Cancer Center - The Chuang Lab </p>
              <p class="subtitle is-6">Bioinformatics researcher - <time datetime="2015-9">February 2015</time> - <time
                      datetime="2019-9">September 2015</time></p>
            </div>
          </div>
          <div class="content">
            <p>Short experience in the field of transcriptomics. During this period I worked remotely for the team
              of Prof. Jeffrey H. Chuang.  </p>
            <ul>
              <li>Analyzed iCLIP data from ENCODE project to study RNA-protein binding sides</li>
            </ul>
            <div class="tags">
              <span class="tag">Python</span>
              <span class="tag">R</span>
              <span class="tag">Perl</span>
              <span class="tag">Bash</span>
              <span class="tag">HPC</span>
              <span class="tag">Biological Data Analysis</span>
              <span class="tag">Statistics</span>
              <span class="tag">Data Visualization</span>
              <span class="tag">Transcriptomics</span>

            </div>
          </div>
        </div>
      </div>
      <div class="card card-gap">
        <div class="card-content">
          <div class="media">
            <div class="media-left">
              <figure class="image is-48x48">
                <img src="img/prbb_logo.png" alt="Placeholder image">
              </figure>
            </div>
            <div class="media-content">
              <p class="title is-4">Barcelona Biomedical Research Park - Computational Genomics Group</p>
              <p class="subtitle is-6">Bioinformatics researcher - <time datetime="2015-9">September 2014</time> - <time
                      datetime="2019-9">February 2015</time></p>
            </div>
          </div>
          <div class="content">
            <p> I contributed in the group of Prof. Eduardo Eyras by bringing my expertise in proteomics (adquired
              during my master thesis) in one of their studies:</p>
            <ul>
              <li>Analyzed and quantified iTRAQ mass spectrometry data from the TCGA project to validate
                findings identified at RNA level</li>
            </ul>
          </div>
        </div>
      </div>
      <div class="card card-gap">
        <div class="card-content">
          <div class="media">
            <div class="media-left">
              <figure class="image is-48x48">
                <img src="img/lund_logo.png" alt="Placeholder image">
              </figure>
            </div>
            <div class="media-content">
              <p class="title is-4">Lund University - Computational Proteomics group</p>
              <p class="subtitle is-6">Master's Student - <time datetime="2015-9">February 2014</time> - <time
                      datetime="2019-9">June 2014</time></p>
            </div>
          </div>
          <div class="content">
            <p>My first practical experience in bioinformatics was in this group leaded by Fredrik Levander.
            I learned the principles of proteomics, MS/MS interpretation and I performed some data analysis:</p>
            <ul>
              <li>Identified and quantified of protein variants using a combination of shotgun proteomics and
                DIA LC-MS</li>
              <li>Applied statistical analysis to the proteins identified for biomarkers discovery</li>
            </ul>

          </div>
        </div>
      </div>
    </div>
  </section>

  <section class="section">
    <div class="container">
      <h1 class="title">Bioinformatic tools</h1>
      <hr />
      <div class="tile is-ancestor">
        <div class="tile is-vertical is-12">
          <div class="tile">
            <div class="tile is-parent">
              <article class="tile is-child notification">
                <p class="title">ipMSDB</p>
                <figure class="image is-3by2">
                  <img class="modal-trigger" data-target="project-1-modal" src="img/carousel/ipmsdb1.png">
                </figure>
              </article>
            </div>
            <div class="tile is-parent">
              <article class="tile is-child notification">
                <p class="title">rBAN</p>
                <figure class="image is-4by3">
                  <img class="modal-trigger" data-target="project-2-modal" src="img/carousel/rban1_low.png">
                </figure>
              </article>
            </div>
          </div>
        </div>
      </div>

      <div class="tile is-ancestor">
        <div class="tile is-vertical is-12">
          <div class="tile">
            <div class="tile is-parent">
              <article class="tile is-child notification">
                <p class="title">NRPro</p>
                <figure class="image is-4by3">
                  <img class="modal-trigger" data-target="project-3-modal" src="img/carousel/nrpro3.png">
                </figure>
              </article>
            </div>
            <div class="tile is-parent">
              <article class="tile is-child notification">
                <p class="title">KFP</p>
                <figure class="image is-4by3">
                  <img class="modal-trigger" data-target="project-4-modal" src="img/carousel/kendrick1.png">
                </figure>
              </article>
            </div>
          </div>
        </div>
      </div>

    </div>
  </section>

  <section class="section">
    <div class="container">
      <h1 class="title">Education / Training</h1>
      <hr />
      <div class="card">
        <div class="card-content">
          <div class="content">
            <p class="title is-4">University of Geneva</p>
            <p class="subtitle is-6">Ph.D. in Bioinformatics - <time datetime="2015-9">September 2015</time> - <time
                datetime="2019-9">Sepemter 2019</time></p>
          </div>
          <div class="content">
            I performed my PhD in a computational research group mainly focused on the development of software tools
            for biological applications, in particular for mass spectrometry analysis. I worked with IT engineers,
            physicist, chemists and biologist. During this period I acquired my development skills. In the "Experience"
            section there are more details about my PhD work.
          </div>
        </div>
      </div>
      <div class="card card-gap">
        <div class="card-content">
          <div class="content">
            <p class="title is-4">University of Barcelona & Polytechnic University of Catalonia</p>
            <p class="subtitle is-6">Master's degree in Biomedical Engineering - <time datetime="2012-9">September 2012</time> - <time
                datetime="2014-9">September 2014</time></p>
          </div>
          <div class="content">
            The master's degree introduced the electrical, mechanical and chemical principals necessary to
            modify/monitor biological systems in order to design tools for diagnosis or treatment of diseases. Some of
            the subjects were:
            <ul>
              <li>Statistical analysis and computation of biomedical data, Modeling and systems physiology,
                Biomaterials, Biomedical Devices, Biomedical Images, Biomedical Signals,
                Tissue engineering and Nanotechnology</li>
            </ul>
          </div>
        </div>
      </div>
      <div class="card card-gap">
        <div class="card-content">
          <div class="content">
            <p class="title is-4">University of Vic - Central University of Catalonia</p>
            <p class="subtitle is-6">Bachelor's degree in Biotechnology - <time datetime="2007-9">September 2007</time> - <time
                    datetime="2011-9">September 2011</time></p>
          </div>
          <div class="content">
            The degree included practical work in the laboratory as well as computational disciplines such
            as programming and statistics. Some of the subjects were:
            <ul>
              <li>Bioinformatics, Biostatistics, Genetic engineering, Microbiology, Proteomics, Bioreactors,
                Immunology and Biochemical engineering</li>

            </ul>
          </div>
        </div>
      </div>
    </div>
  </section>

  <section class="section">
    <div class="container">
      <h1 class="title">Contact</h1>
      <hr />
      <nav class="level">
        <div class="level-item has-text-centered">
          <div>
            <a href="https://github.com/ericart89" class="icon is-large">
              <i class="fab fa-2x fa-github"></i>
            </a>
            <p class="heading"><a href="https://github.com/ericart89">github.com/ericart89</a></p>
          </div>
        </div>

        <div class="level-item has-text-centered">
          <div>
            <a href="https://bitbucket.org/EmmaRicart" class="icon is-large">
              <i class="fab fa-2x fa-bitbucket"></i>
            </a>
            <p class="heading"><a href="https://bitbucket.org/EmmaRicart">bitbucket.org/EmmaRicart</a></p>
          </div>
        </div>

        <div class="level-item has-text-centered">
          <div>
            <a href="mailto:emma.ricart89@gmail.com" class="icon is-large">
              <i class="fas fa-2x fa-envelope"></i>
            </a>
            <p class="heading"><a href="mailto:emma.ricart89@gmail.com">emma.ricart89@gmail.com</a></p>
          </div>
        </div>
        <div class="level-item has-text-centered">
          <div>
            <a href="https://www.linkedin.com/in/emma-ricart-phd-215028137" class="icon is-large">
              <i class="fab fa-2x fa-linkedin-in"></i>
            </a>
            <p class="heading"><a href="https://www.linkedin.com/in/emma-ricart-phd-215028137">linkedin</a></p>
          </div>
        </div>
      </nav>
    </div>
  </section>

  <!-- Modals -->
  <div id="project-1-modal" class="modal">
    <div class="modal-background"></div>
    <div class="modal-card">
      <header class="modal-card-head">
        <p class="modal-card-title">ipMSDB</p>
        <button class="delete" aria-label="close"></button>
      </header>
      <section class="modal-card-body">

        <div id="project-1-modal-carousel" class="carousel">
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/ipmsdb1.png">
          </figure>
        </div>

        <div class="content">
          <p>Immunopeptidomics database with more 1.000.000 HLAI/HLAII peptides. The data was retrieved from thousands
            of mass spectrometry experiments and analyzed in an HPC cluster with a nextflow pipeline that included tools
            such as Fragpipe and MixMHC. Ongoing project.
          </p>
          <div class="tags">
            <span class="tag">PostgreSQL</span>
            <span class="tag">Django</span>
            <span class="tag">Python</span>
            <span class="tag">Nextflow</span>
          </div>

        </div>

      </section>
      <footer class="modal-card-foot">
        <button class="button is-success">Close</button>
      </footer>
    </div>
  </div>

  <div id="project-2-modal" class="modal">
    <div class="modal-background"></div>
    <div class="modal-card">
      <header class="modal-card-head">
        <p class="modal-card-title">rBAN</p>
        <button class="delete" aria-label="close"></button>
      </header>
      <section class="modal-card-body">

        <div id="project-2-modal-carousel" class="carousel">
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/rban1_low.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/rban2.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/rban3.png">
          </figure>
        </div>

        <div class="content">

          <p>rBAN (retroBiosynthetic Analyisis of Nonribosomal peptides) is  a computational tool designed to predict
            the monomeric graph of nonribosomal peptides from their atomic structure. To achieve that
            the chemical structure of the input compounds (SMILES format) is first converted into a graph representation.
            Then, certain bonds are mapped using a substructure search algorithm. Finally, these bonds are disconnected
            and the resulting fragments are matched against a monomers database in order to identify the moieties.</p>

          <div class="tags">
            <span class="tag">Java</span>
            <span class="tag">Javascript</span>
            <span class="tag">d3.js</span>
            <span class="tag">Biochemistry</span>
            <span class="tag">Graph</span>
            <span class="tag">Substructure search</span>
          </div>

        </div>

      </section>
      <footer class="modal-card-foot">
        <button class="button is-success">Close</button>
      </footer>
    </div>
  </div>

  <div id="project-3-modal" class="modal">
    <div class="modal-background"></div>
    <div class="modal-card">
      <header class="modal-card-head">
        <p class="modal-card-title">NRPro</p>
        <button class="delete" aria-label="close"></button>
      </header>
      <section class="modal-card-body">
        <div id="project-3-modal-carousel" class="carousel">
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/nrpro1.jpeg">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/nrpro2.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/nrpro3.png">
          </figure>
        </div>

        <div class="content">
          <p>NRPro is a web application for the identification and annotation of tandem mass
            spectra from peptidic natural products. That involved the implementation of a
            fragmentation model mimicking the breakages of these compounds in the mass
            spectrometer. The generated fragmentation trees are matched and scored against the
            experimental spectra. The database behind the software is the result of merging
            compounds from Norine, ChEBI and the Natural Products Atlas.</p>
          <div class="tags">
            <span class="tag">Java</span>
            <span class="tag">Tomcat</span>
            <span class="tag">MongoDB</span>
            <span class="tag">AngularJS</span>
            <span class="tag">Mass spectrometry</span>
          </div>
        </div>

      </section>
      <footer class="modal-card-foot">
        <button class="button is-success">Close</button>
      </footer>
    </div>
  </div>

  <div id="project-4-modal" class="modal">
    <div class="modal-background"></div>
    <div class="modal-card">
      <header class="modal-card-head">
        <p class="modal-card-title">KFP</p>
        <button class="delete" aria-label="close"></button>
      </header>
      <section class="modal-card-body">
        <div id="project-4-modal-carousel" class="carousel">
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/kendrick2.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/kendrick1.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/kendrick3.png">
          </figure>
        </div>

        <div class="content">
          <p>KFP (Kendrick Formula Predictor) is a web application to predict peptidic natural products molecular formula
            out of a given monoisotopic mass using the Kendrick Mass defect (KMD). The approach consists in using a
            target database to create the KMD plot from which molecular formula can be predicted.</p>
          <div class="tags">
            <span class="tag">Java</span>
            <span class="tag">Javascript</span>
            <span class="tag">d3.js</span>
          </div>
        </div>

      </section>
      <footer class="modal-card-foot">
        <button class="button is-success">Close</button>
      </footer>
    </div>
  </div>
<!--
  <div id="project-5-modal" class="modal">
    <div class="modal-background"></div>
    <div class="modal-card">
      <header class="modal-card-head">
        <p class="modal-card-title">Project title</p>
        <button class="delete" aria-label="close"></button>
      </header>
      <section class="modal-card-body">
        <div id="project-5-modal-carousel" class="carousel">
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/1.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/2.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/3.png">
          </figure>
        </div>

        <div class="content">
          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
            dolore
            magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
            commodo
            consequat.</p>
          <div class="tags">
            <span class="tag">angular</span>
            <span class="tag">mongo</span>
            <span class="tag">typscript</span>
            <span class="tag">node</span>
          </div>
        </div>

      </section>
      <footer class="modal-card-foot">
        <button class="button is-success">Close</button>
      </footer>
    </div>
  </div>

  <div id="project-6-modal" class="modal">
    <div class="modal-background"></div>
    <div class="modal-card">
      <header class="modal-card-head">
        <p class="modal-card-title">Project title</p>
        <button class="delete" aria-label="close"></button>
      </header>
      <section class="modal-card-body">
        <div id="project-6-modal-carousel" class="carousel">
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/1.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/2.png">
          </figure>
          <figure class="image is-4by3 carousel-cell">
            <img src="img/carousel/3.png">
          </figure>
        </div>

        <div class="content">
          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
            dolore
            magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
            commodo
            consequat.</p>
          <div class="tags">
            <span class="tag">angular</span>
            <span class="tag">mongo</span>
            <span class="tag">typscript</span>
            <span class="tag">node</span>
          </div>
        </div>

      </section>
      <footer class="modal-card-foot">
        <button class="button is-success">Close</button>
      </footer>
    </div>
  </div>
-->
  <footer class="footer">
    <div class="content has-text-centered">
      <p>
        <a href="https://github.com/ericart89/devfolio" target="_blank" class="icon has-text-primary">
          <i class="fab fa-github"></i> <strong>emma_portfolio</strong></a>
      </p>
    </div>
  </footer>


  <script src="js/bundle.js"></script>
</body>

</html>